<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>767</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>961</width>
    <height>767</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>961</width>
    <height>767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3DViewer_v2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Scene" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_file_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>700</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>881</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_zoom">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom by</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_x">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate around X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_z">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate around Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_y">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate around Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_move_x">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move by X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_move_y">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move by Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_decorative">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Model Display</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_rotate_x">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_rotate_z">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_rotate_y">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_move_y">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_move_x">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_apply">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_line_color">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Line Color</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zoom_spinbox">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.500000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertex">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vertex number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_lines">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lines number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_amount">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>640</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="edge_amount">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>670</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_move_z">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_move_z">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move by Z</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_line_thickness">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_line_thickness">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Thickness</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_solid">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_line</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_dashed">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashed</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_line</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_point_color">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Dot Color</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_point_size">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_point_size">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Point size</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_point_none">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>490</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_point</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_point_square">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>490</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_point</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_point_circle">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>490</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_point</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_bg_color">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>520</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Background Color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radi_projection_central">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_projection</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radi_projection_parallel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>570</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radio_projection</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_screenshot">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_gif">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QWidget</extends>
   <header>scene.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radio_line"/>
  <buttongroup name="radio_point"/>
  <buttongroup name="radio_projection">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
